build_tools/buildkite/scripts/update_pipeline_configurations.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python3
```

Comment:
I assume this file is mostly just renamed from `update_pipeline_configuration.py`?

Reply:
Yeah. Git move detection really isn't great :-/